src/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,289 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@rkemmy, you might agree that while this snapshot is easy for the computer to read, it is very difficult for the human eye. In addition, at close to 300 lines, the snapshot is too big for such a small component as our `App` currently is 🤔; what will happen once we start having bigger components? I guess the computer will also start finding it difficult to parse the snapshots quickly because they'll be massive in size! Consequently, our tests will take long to run and they may even fail.

Luckily, there is a solution to this. We should consider a better snapshot serializer. Please look up how to plug in `enzyme-to-json` snapshot serializer using the `setupTests.js` and we'll be safe. 

After that, you can delete this snapshot and rerun the tests to get a better snapshot generated; you're likely see the difference.

Reply:
I have made the changes and the snapshot is considerably smaller than the previous one. Kindly review and let me know if there are any more changes to be made.

Reply:
@rkemmy, if a dependency is only useful during development work such as testing dependencies and package bundling dependencies like webpack, babel dependencies, code syntax linters etc, then it's a development dependency. Else if a dependency will be needed for the application to be able to run at all in a production environment, e.g. react, redux I think this can answer better https://stackoverflow.com/a/46060407/6951110.